hips/tiles/surveys.py

```diff
@@ -58,6 +58,10 @@ def parse(cls, text: str) -> 'HipsSurveyPropertiesList':
         surveys = list()
         for raw_survey in text.split('\n\n'):
             surveys.append(HipsSurveyProperties.parse(raw_survey).properties)
+        return cls(surveys)
+
+    @property
+    def table(self) -> Table:
```

Comment:
Every public property needs at least a one-line docstring.
And there should be a test covering everything, i.e. in this case for the table.

Also -- I think this way of creating a Table isn't useful. Can you do `table.info()` to show what it contains? Is it a single column? You can't do any selection like that, no?

Probably what you want is to have a table with one row per survey, and the properties as columns.
Can you try `table = Table(rows=rows)` and pass as `rows` a Python list of `OrderedDict` objects?
That should populate the table in a useful way, i.e. with multiple columns and rows as I mentioned above.

If this doesn't work, you could also remove the table property here and then add that in a future PR.